### PR TITLE
close incident if duplicate playbook (ML)

### PR DESCRIPTION
**Pull request by erezh31**
fixes:
https://github.com/demisto/etc/issues/10402

### CODE REVIEW

**Comment by erezh31** [comment]
@meirwah plz review

File: Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml [hunk 1]
```diff
@@ -0,0 +1,252 @@
+id: close_incident_if_duplicate_found_ml
+version: -1
+name: Close incident if duplicate found ML
```
Comment (meirwah) [inline Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml]: DeDup incidents - ML

File: Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+- key: closeThreshold
+  value:
+    simple: "0.75"
+  required: true
+  description: Candidate with score above this threshold will close the investigation
+    automatically, and mark as duplicate to the current incident.
+outputs:
+- contextPath: foundDuplicates
+  description: found any duplicate incident?
+  type: unknown
```
Comment (meirwah) [inline Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml]: not bool?

File: Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+- contextPath: duplicateCandidate
+  description: the duplicate top candidate
+  type: unknown
```
Comment (meirwah) [inline Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml]: not string?

File: Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+      }
+  "1":
+    id: "1"
+    taskid: becd6edd-f25f-43a4-8099-9c55a32210dd
+    type: regular
+    task:
+      id: becd6edd-f25f-43a4-8099-9c55a32210dd
+      version: -1
+      name: Find duplicate incidents
+      description: Find duplicate incidents by machine-learning
```
Comment (meirwah) [inline Playbooks/playbook-Close_incident_if_duplicate_found_ML.yml]: Find duplicate incidents using machine-learning

**Pull request merged**